spec/models/step_by_step_page_spec.rb

```diff
@@ -280,6 +279,14 @@
       step_by_step_page.mark_draft_updated
       expect(step_by_step_page.scheduled_for_publishing?).to be false
     end
+
+    it 'clears the scheduled status after unscheduling' do
```

Comment:
This test isn't needed as it's testing the same things as `it 'is not scheduled for publishing if a draft has not been saved'` and `it 'is not scheduled for publishing if scheduled_at is not present'`.

I think the most appropriate place for a test like this would be in a controller test.